apps/flame-defi/app/pool/components/transaction-summary/transaction-summary.types.ts

```diff
@@ -31,5 +49,5 @@ export type TransactionSuccessProps = {
 };
 
 export type TransactionFailedProps = {
```

Comment:
There is another `TransactionFailedProps` except it has `message` string instead of `error`. Can we get rid of one of them?